metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -629,6 +629,9 @@ def _ensure_valid_gms_url_acryl_cloud(url: str) -> str:
         url = url.replace("http://", "https://")
     if url.endswith("acryl.io"):
         url = f"{url}/gms"
+    elif url.endswith("acryl.io/"):
+        url = f"{url}gms"
+
```

Comment:
we actually don't need this - `fixup_gms_url` is what calls `_ensure_valid_gms_url_acryl_cloud`, and already removes the trailing slash

that said, it probably would be good to add some tests around this so we don't have a regression

Reply:
@chriscollins3456 `test_cli_utils.py` has some unit tests that you should probably add to if you found cases where it wasn't doing what it was supposed to do.